### ISSUE TITLE
Support non-primitive / complex values such as list

### ISSUE THREAD

**Issue by mdutoo** [erocci/erocci#45]
Supporting OCCI attributes with non-primitive / complex values such as list would be useful.

For instance in the LDaaS use case, Ozwillo Datacore Linked Data projects can be configured with a "dcmp:frozenModelNames" field that is a list of model names. But at worse I'll uglily use json with json...

1. non-primitive / complex attributes are not well defined in the OCCI specification, what is the status there ? where can it be followed (web links) ?
2. same in erocci ?
3. same in erocci-dbus-java ?

(if I can, I'll update some of those points)

**Comment by jeanparpaillon** [comment]
I totally agree with you on the need. BTW, 1.2 specs allows for attributes to be 'Object' in JSON, but without any means to define it in schemas.

As any OCCI specs improvement, complex basic datatypes sould have a definition in Core, then renderings.

Let me first writing a proposal for next OCCI meeting (april).

**Comment by mdutoo** [comment]
Yeah, it's weird (to say the least) that renderings define the type system...
